android/hello_sdl_android/src/main/res/layout/activity_main.xml

```diff
@@ -1,16 +1,81 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:paddingBottom="@dimen/activity_vertical_margin"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="com.sdl.hellosdlandroid.MainActivity" >
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent">
 
     <TextView
```

Comment:
@santhanamk this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?

Reply:
@kostyaBoss I still see this issue.  Are you able to export all of the hard coded strings (values for `android:text` and `android:hint`) and put them in the `strings.xml` file?